### PR TITLE
New package: act-0.2.24

### PR DESCRIPTION
**Pull request by kwshi**
First time submitting a new package, I may be doing a number of things wrong, let me know what needs to be fixed!

### CODE REVIEW

File: srcpkgs/act/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'act'
+pkgname=act
+version=0.2.17
```
Comment (ericonr) [inline srcpkgs/act/template:3]: 0.2.23 is out

**Review by ericonr — COMMENTED** [review]
This looks interesting :)

File: srcpkgs/act/template (hunk 1 above)
Comment (kwshi) [reply srcpkgs/act/template]: Updated!

**Comment by meator** [comment]
Hi. 0.2.24 is out. Is this PR still active?

**Comment by kwshi** [comment]
> Hi. 0.2.24 is out. Is this PR still active?

@meator just updated the PR to 0.2.24.  :)

**Pull request closed without merging**